hdl/XilinxBuildScript.tmpl

```diff
@@ -56,10 +56,10 @@ report_ip_status
 {{ end }}
 
 {{ range .Constrs }}
-read_xdc "{{ . }}"
+	read_xdc {{ if $outOfContext }} -mode out_of_context {{ end }} "{{ . }}"
```

Comment:
@lj This does not work if I put `{{ .OutOfContext }}` directly. Any idea why?

Reply:
Because it's in a loop. The meaning of `.` in this scope is no longer the structure you passed to the script, but the current element of the `.Constrs` slice.

Reply:
I see, thanks!